### DISCUSSION TITLE
Sticky navbar

### DISCUSSION THREAD

**Discussion by LITUATUI** [jpanther/congo#470] [Ideas]
Please add the option to make the navbar sticky, or make it default.

Thanks!

**Comment by max-alletsee** [comment]
Hey,

I know that the topic is already a bit older, but it's luckily quite easy to solve this via a custom layout. For the sticky header on my website, I simply copied the `hamburger.html` from the header partials ([this file](https://github.com/jpanther/congo/blob/dev/layouts/partials/header/hamburger.html)) into a new `custom.html` in my `layouts` folder and added the following to the first line: `sticky top-0 z-50` 

... so it looks something like this:

`<header class="py-6 font-semibold text-neutral-900 dark:text-neutral print:hidden sm:py-10 sticky top-0 z-50">`

(Here is an [example result](https://www.klimafreundlicher-kochen.de/))

Hope this helps!
